Makefile

```diff
@@ -34,20 +34,25 @@ dev: build/manifest.json .pydeps
 docker:
 	git archive HEAD | docker build -t hypothesis/hypothesis:$(DOCKER_TAG) -
 
-# Run docker image built with `docker` task
+# Run docker image built with `docker` task.
+# This command is intended only for use in development and assumes that the
+# service containers (Postgres, Elasticsearch etc.) have been run using
+# docker-compose.
 .PHONY: run-docker
 run-docker:
-	$(eval RABBITMQ_CONTAINER ?= rabbitmq)
+	$(eval RABBITMQ_CONTAINER ?= rabbit)
```

Comment:
Also I don't think there's a need to change the name from rabbitmq->rabbit. I think you can just leave it as is.

Reply:
Hmm ... I suppose `_HOST` would be more accurate than `_CONTAINER` now. I still think it is useful to be able to customize in case you want to test the Docker container with a different service container for some reason, although perhaps in that case one might just make a temporary edit to the `Makefile`.